frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -446,15 +449,25 @@ const QueueItem: any = ({
     value: device.internalId,
   }));
 
-  const patientFullName = displayFullName(
+  const patientFullNameLastFirst = displayFullName(
     patient.firstName,
     patient.middleName,
     patient.lastName
   );
 
+  const patientFullName = displayFullName(
```

Comment:
Did the designers really want the name orders reversed here?

Reply:
🤷‍♂️ good question. I just assumed it'd be nice to get the first name in either the `Last, First Middle` or `First Middle Last` formats.